units/patch_intent.go

```diff
@@ -1026,3 +1016,17 @@ func (j *patchIntentProcessor) sendGitHubErrorStatus(patchDoc *patch.Patch) {
 
 	j.AddError(update.Error())
 }
+
+// sendGitHubStatusMessage sends a successful status to Github with the given message.
+func (j *patchIntentProcessor) sendGitHubStatusMessage(patchDoc *patch.Patch, msg string) {
+	update := NewGithubStatusUpdateJobWithMessage(
+		evergreenContext,
+		patchDoc.GithubPatchData.BaseOwner,
+		patchDoc.GithubPatchData.BaseRepo,
+		patchDoc.GithubPatchData.HeadHash,
+		msg,
+	)
+	update.Run(nil)
```

Comment:
The [context docs](https://pkg.go.dev/context@go1.20.1#pkg-overview) say to never pass a nil context because it has the potential for bad things to happen, especially as future-proofing if one day the context does get used. Can we just pass the `patchIntentProcessor` job's context into this?